### ISSUE TITLE
Invalid timestamps generated

### ISSUE THREAD

**Issue by aramlang** [m-bain/whisperX#153]
So I am running this:
```
whisperx --model large-v2 --language he --align_model imvladikon/wav2vec2-xls-r-1b-hebrew --vad_filter ./bereshit01.wav
```
The resulting `srt` file has completely non-sensical timestamps. Any idea what I am doing wrong?
The 0.720s is basically when the first word finishes, not the 7th!
```
1
00:00:00,000 --> 00:00:00,120
בראשית
2
00:00:00,140 --> 00:00:00,200
ברא
3
00:00:00,220 --> 00:00:00,340
אלוהים
4
00:00:00,360 --> 00:00:00,400
את
5
00:00:00,420 --> 00:00:00,540
השמיים
6
00:00:00,560 --> 00:00:00,620
ואת
7
00:00:00,640 --> 00:00:00,720
הארץ
```

**Comment by aramlang** [comment]
The audio file.
[bereshit01.zip](https://github.com/m-bain/whisperX/files/11074782/bereshit01.zip)

Couldn't figure out what the issue was, I am not too versed in this field yet, but those timestamps a complete bullock. 
Unless we get some resolution here, it would seem https://github.com/linto-ai/whisper-timestamped might give me what I am looking for.

**Comment by kurianbenoy-sentient** [comment]
@aramlang if you know the language you are working on, please specify the language as well with attribute (--language). As far I know, Whisper supports the languages in given list:

https://github.com/m-bain/whisperX/blob/main/whisperx/tokenizer.py#L10-L110

While whisperx supports even fewer languages due to necessity of phenome model for forced alignment. I think the default model supports only the list of given languages:

https://github.com/m-bain/whisperX/blob/main/whisperx/alignment.py#L26-L40

**Comment by mayeaux** [comment]
In my testing the re-alignment is very hit or miss. I was trying with some Serbian models and I had similar output to you where the timestamps per segment all because squished within a half second or so. Actually, the English model worked best for Serbian output (decent chance that the 3 models for Serbian I tried were not great), but even in cases where the English model had issues it would break the timestamps pretty badly.

**Comment by aramlang** [comment]
I am very new to the field of NLP. So I am not even sure what I am saying makes sense. But based on interactions with ChatGPT, it would seem that the problem is that the `imvladikon/wav2vec2-xls-r-1b-hebrew` model might not have "log-probabilities". Does that make sense to anybody here? 
Is there a way to rebuild those models to include that "log-probabilities" information?

**Comment by aramlang** [comment]
Issue fixed by #216
